src/main/java/org/mockito/internal/session/DefaultMockitoSessionBuilder.java

```diff
@@ -0,0 +1,33 @@
+package org.mockito.internal.session;
+
+import org.mockito.MockitoSession;
+import org.mockito.internal.framework.DefaultMockitoSession;
+import org.mockito.internal.util.ConsoleMockitoLogger;
+import org.mockito.quality.Strictness;
+import org.mockito.session.MockitoSessionBuilder;
+
+public class DefaultMockitoSessionBuilder implements MockitoSessionBuilder {
+
+    private Object testClassInstance;
+    private Strictness strictness;
+
+    @Override
+    public MockitoSessionBuilder initMocks(Object testClassInstance) {
```

Comment:
OK for `initAnnotated`, but what about `processAnnotations`.

> @bric3, I'd like to move on with your original idea, e.g. session().initMocks(this). If we want to change initMocks in both, session and MockitoAnnotations, let's move it to a separate ticket.

That's an option bu in the meantime we introduce an API that could be deprecated. And in the next PR, it's probably gonna be discarded because the API has just been introduced in mockitosession and we won't deprecate something that just entered the scene. We rarely change incubating APIs. At least we could mention it in the javadoc.